### PR TITLE
Adjust onboarding importer navigation and styles

### PR DESCRIPTION
**Pull request by bogiii**
#### Changes proposed in this Pull Request

Changes introduce header navigation with proper links. Following Figma design, there are "Back" and "I don't have a site address" buttons.

The scanning route has been removed from the flow and now it is a part of the capture screen.

Other than that, there are a few styling adjustments.

#### Testing instructions

Manual testing:
- go to `/start/importer/`
- enter any valid URL and hit the return key
- `scanning` screen will be shown, but this time without redirecting to the scanning route
- after the scanning process, the ready page will be shown
- try hitting header buttons (the initial back button will not work since we manually come to the page)

#### Screenshots

Desktop
Screen Capture on 2021-10-25 at 23-41-47

Mobile

Related to
- [Figma design](https://www.figma.com/file/TAK5FpwTvfbENJDfHc7Aq1/Site-Importing?node-id=1930%3A21706)

### CODE REVIEW

File: client/signup/steps/import/capture/index.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+	return (
+		<div className="import-layout__center">
+			<div className="capture__content">
+				<input
+					className="capture__input"
+					autoComplete="off"
+					autoCorrect="off"
+					spellCheck="false"
+					placeholder={ __( 'Enter your site address' ) }
```
Comment (a8ci18n) [inline client/signup/steps/import/capture/index.tsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Enter your new site address' )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/gutenboarding-import-navigation)

File: client/signup/steps/import/list/index.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+const ListStep: React.FunctionComponent = () => {
+	const { __ } = useI18n();
+	const history = useHistory();
+
+	return (
+		<>
+			<div className={ 'import-layout list__wrapper' }>
+				<div className={ 'import-layout__column' }>
+					<div className="import__heading">
+						<Title>{ __( 'Import your content from another platform' ) }</Title>
```
Comment (a8ci18n) [inline client/signup/steps/import/list/index.tsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Import content from another website or platform.' )` *ES Score: 7*

File: client/signup/steps/import/ready/index.tsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+									website: convertToFrendlyWebsiteName( website ),
+									platform,
+								}
+							),
+							{ strong: createElement( 'strong' ) }
+						) }
+					</SubTitle>
+
+					<div className="import__buttons-group">
+						<NextButton>{ __( 'Import your content' ) }</NextButton>
```
Comment (a8ci18n) [inline client/signup/steps/import/ready/index.tsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Import Your Content' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/gutenboarding-import-navigation)

File: client/signup/steps/import/ready/platform-details.tsx [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+						{ createInterpolateElement( __( 'Things we <strong>can</strong> import:' ), {
+							strong: createElement( 'strong' ),
+						} ) }
+					</p>
+					<ul className={ 'import__details-list' }>
+						<li>
+							<Icon size={ 20 } icon={ check } /> { __( 'Blog posts' ) }
+						</li>
+						<li>
+							<Icon size={ 20 } icon={ check } /> { __( 'Static pages' ) }
```
Comment (a8ci18n) [inline client/signup/steps/import/ready/platform-details.tsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'All static pages' )` *ES Score: 7*

**Comment by vishnugopal** [comment]
Most tests are failing because of this: 
```
There is a merge conflict with trunk. Rebase on trunk to resolve this problem.
```

Think we can safely ignore this as the source branch is different.

**Review by vishnugopal — APPROVED** [review]
Approving, everything seems to work. Great job! :)

Leaving a few comments here for future PRs:
1. Ollie's feedback about the validation being active when the user starts typing.
2. The site previews on a Retina display look pixelated. Is this expected, and anything we can do to improve?
3.  I noticed we have a `/start/import` route already. This is slightly confusing. It may be wise to rename this to `/start/existing-site` to match the user-facing copy.

CleanShot 2021-10-26 at 10 01 39@2x

**Comment by pento** [comment]
> The site previews on a Retina display look pixelated. Is this expected, and anything we can do to improve?

Adding `?scale=2` to the mShots URL improves the quality. Usually not needed with tiny screenshots, but it's useful here. 🙂

> I noticed we have a /start/import route already. This is slightly confusing. It may be wise to rename this to /start/existing-site to match the user-facing copy.

Is `/start/import` used any more? We really should be aiming to deprecate it.

**Comment by vishnugopal** [comment]
> Is /start/import used any more? We really should be aiming to deprecate it.

Agreed! I'll have a conversation with @simison about this. Nevertheless, my point about renaming route to match the user-facing copy remains. It makes future grepping easier :)

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/6907218

Thank you @bogiii for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/6907218).

**Pull request merged**